student-content/templates/gitea/console-link.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: console.openshift.io/v1
```

```suggestion
---
apiVersion: console.openshift.io/v1
```

I like yaml separators 🙈